packages/babel-types/test/validators.js

```diff
@@ -0,0 +1,30 @@
+var t = require("../lib");
+var assert = require("assert");
+var parse = require("babylon").parse;
+
+suite("validators", function () {
```

Comment:
This was long ago - got a linting error because of the merge and had to change it to let